src/names/Danmark.txt

```diff
@@ -0,0 +1,511 @@
+Pernille
+Rasmus
+Heidi
+annette
+Jacob
+Anette
+Inger
+Finn
+Birgitte
+Dorthe
+Ulla
+Henning
+Bjarne
+Svend
+Birgit
+Kenneth
+Lisbeth
+Karin
+Knud
+Bo
+Mogens
+Dorte
+Birthe
+Else
+Leif
+Trine
+Kristian
+Eva
+Jytte
+Line
+Kurt
+Kasper
+Jakob
+Lis
+Vibeke
+Lise
+Karina
+Preben
+Frank
+Mikkel
+Arne
+Marie
+Tove
+Jane
+Klaus
+Anja
+Rene
+Malene
+Lasse
+Lotte
+Daniel
+Karsten
+Jonas
+Stine
+Simon
+Bodil
+Grethe
+Linda
+Dennis
+Tine
+Kaj
+Ellen
+Julie
+Tommy
+Henriette
+Andreas
+Jeanette
+Nina
+Johnny
+Palle
+Ann
+Anita
+Gert
+Katrine
+Signe
+Bettina
+Ruth
+Anni
+Dan
+Mona
+Tom
+Karl
+Steffen
+Benny
+Sanne
+Merete
+Casper
+Mia
+Carl
+Ida
+Laila
+Ib
+Ditte
+Kjeld
+Aase
+Ingrid
+Tanja
+Mikael
+Federik
+Maja
+Stig
+Alice
+Britta
+Kristina
+Erling
+Nanna
+Margit
+Berit
+Betina
+Lena
+Sara
+Joan
+Mathias
+Sonja
+Rune
+Connie
+Jeppe
+Ove
+Keld
+Jonna
+Peder
+Janne
+Rene
+Vagn
+Nicolai
+Helene
+Ulrik
+Gerda
+Birte
+Elisabeth
+David
+Jannie
+Susan
+Troels
+Sofie
+Inga
+Rita
+Nikolaj
+Stefan
+Carina
+Sarah
+Cecilie
+Emil
+Irene
+Michelle
+Gunnar
+Lisa
+Randi
+Robert
+Alex
+Vivi
+Mark
+Johannes
+Janni
+Egon
+Ivan
+Mie
+Kent
+Helge
+Christine
+Liselotte
+Annelise
+Edith
+Iben
+Marlene
+Katja
+Tage
+Lillian
+Patrick
+Uffe
+Kristine
+Kathrine
+Johan
+Britt
+Winnie
+Sven
+Sandra
+Astrid
+Grete
+Aage
+Jimmy
+Elsebeth
+Elin
+Diana
+Alexander
+Solveig
+Tobias
+Tonny
+Majbritt
+Villy
+Esben
+Conny
+Benjamin
+Holger
+Paul
+Lea
+Laura
+Ingelise
+Maiken
+Asger
+Nils
+Kate
+Yvonne
+Aksel
+Sune
+Esther
+Verner
+Gurli
+Caroline
+Lisbet
+Ane
+Sebastian
+Simone
+Henry
+Jes
+Sten
+Philip
+Ebbe
+Frede
+Tim
+Henny
+Johanne
+Erna
+Birger
+Bjarke
+Gudrun
+Christoffer
+Annemette
+Chris
+Lilian
+Freddy
+Kamilla
+Elly
+Jon
+Max
+Leo
+Helga
+Doris
+Viggo
+Maj-Britt
+Lilly
+Jimmy
+Kirstine
+Lissi
+Magnus
+Kai
+Ina
+Ella
+Kristoffer
+Thorkild
+Helen
+Emma
+Nick
+Nadia
+Orla
+Bruno
+Vera
+Ernst
+Ketty
+Annika
+Richard
+Harry
+Ninna
+Majken
+Nicolaj
+Jeanne
+Peer
+Anny
+Nicklas
+Sren
+Lise-lotte
+Mohammad
+Conni
+Rolf
+Monica
+Mathilde
+Annemarie
+Danny
+Ejvind
+Bitten
+Vivian
+Margrethe
+Otto
+Ali
+Willy
+Jack
+Jenny
+Maibritt
+Mary
+Jim
+Ronni
+Ilse
+Betty
+Emilie
+Lykke
+Axel
+Kenn
+Eigil
+Majbrit
+Sine
+Marc
+Hanna
+Asta
+Sussi
+Cathrine
+Mike
+Torsten
+Ester
+Sabrina
+Vita
+Lilli
+Mariann
+Lennart
+Minna
+Sidsel
+Jannik
+Dorrit
+Christa
+Jess
+Agnete
+Gunhild
+Anton
+Sigrid
+Amalie
+Helena
+Herids
+Ken
+Kevin
+Ebba
+Heine
+Christel
+Åse
+Edel
+Ejner
+Glenn
+Rigmor
+Judith
+Jeannette
+Agnes
+Ejnar
+Tenna
+Niklas
+Tony
+Elna
+Gunner
+Rosa
+Sabine
+Leon
+Frits
+Stina
+Irma
+Rie
+Hugo
+Joachim
+Frode
+Lissy
+Dagny
+Christopher
+Stephanie
+Martha
+Bendt
+Tonni
+Ronnie
+Georg
+Harald
+Ingeborg
+Hardy
+Olga
+Mai
+William
+Adam
+Marian
+Oliver
+Sissie
+Kaja
+Tomas
+Dina
+Jeanett
+Josefine
+Amanda
+Theis
+Teddy
+Lonnie
+Kresten
+Thea
+Rebecca
+Monika
+Thor
+Lærke
+Mille
+Maren
+Sussie
+Janus
+Steven
+Andrea
+Gorm
+Anker
+Alfred
+Elise
+Jonathan
+Solvejg
+Kamma
+Ulrich
+Victor
+Paw
+Maj
+Kåre
```

Comment:
Please run the cleanup script to avoid impossible Minecraft names

Reply:
Hi, i've tried to run the cleanup script, i'm not sure why but it's not working for me, if you're able to help me get it to work I would gladly do so, as you can see I also said this to this person here https://github.com/LabyMod/og-names/pull/9#issuecomment-996184273

Reply:
I have done as requested of you now, i think so.